common/app/templates/inlineJS/nonBlocking/detectAdblock.scala.js

```diff
@@ -0,0 +1,24 @@
+try {
+    ((document, window) => {
+        var ad = document.createElement('div');
+        ad.style.position = 'absolute';
+        ad.style.left = '-9999px';
```

Comment:
I thought I'd committed a fix to the -9999px hack - evidently not. I believed that some older user agents would reserve a large canvas in memory, and this could cause scroll jank.